src/components/authentication/requestAccess.html

```
@@ -23,7 +24,7 @@ <h1 class="f2 pivot-dark-blue fw6 mb4 mt0">
                     </div>
                     <div class="mt3">
                         <label class="db fw4 lh-copy f6">{{ 'requestAccess.form.email' | t }}</label>
-                        <input class="pa1 input-reset ba bg-transparent w-100" type="email" required ng-model="requestData.email">
+                        <input class="pa1 input-reset ba bg-transparent w-100" type="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,63}$" required ng-model="requestData.email" >
```

https://github.com/angular/angular.js/blob/master/src/ng/directive/input.js#L27 is what Angular is using. I feel like this issue is something that always comes up in projects :) It must support an intranet email server or something like `my-email@mycompany`. I'd say it's fine to just let the server throw the error if it wants it stricter. But if you like your regex or want to ammend Angular's we could do that to give them better feedback on the client. I was just using `asdf@asdf` to see what happened in the case of a server error :)

---

Yah exactly, its like that as sdfs@asdfsd is a valid email address or sddsf@[127.0.0.1] etc..  yah i think keeping the pattern is fine, prevents unneeded trip to server if someone accidentally forgets the domain extension.